src/Adapter/MSTestAdapter.PlatformServices/MSTestAdapter.PlatformServices.csproj

```diff
@@ -51,6 +51,11 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.TestPlatform.MSTestAdapter.UnitTests" Key="$(VsPublicKey)" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(UwpMinimum)' OR '$(TargetFramework)' == '$(ModernUwpMinimum)' OR '$(TargetFramework)' == '$(WinUiMinimum)' ">
+    <!-- CVE-2019-0981, CVE-2019-0980, CVE-2019-0657 -->
+    <PackageReference Include="System.Private.Uri" />
```

Comment:
I think there was a recent fix on NuGet side to avoid these false positives. It would be good if you can double check whether this is still needed or not.

Reply:
Will do on a separate PR.

Reply:
It's already removed on main so yep this needs to be dropped.